tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -148,6 +148,25 @@ def controlled_allocator(size):
 
         assert not cupy.any(cupy.isnan(percentiles))
 
+    # See gh-4607
+    # "Magic" values used in this test were empirically found to result in
+    # non-monotonicity for less accurate linear interpolation formulas
+    @testing.for_orders((-29, -53, -207, -16373, -99999,), name='magic_value')
```

Comment:
`testing.for_orders` is a decorator to parameterize tests with order of ndarray. Could you use `testng.parameterize` instead?

Reply:
Oh yes! Sorry about that. Just pushed a correction.